wallet/src/Pos/Wallet/Web/Pending/Submission.hs

```diff
@@ -116,16 +104,28 @@ submitAndSavePtx PtxSubmissionHandlers{..} enqueue ptx@PendingTx{..} = do
     when walletTxCreationDisabled $
         throwM $ InternalError "Transaction creation is disabled by configuration!"
 
-    ack <- submitTxRaw enqueue _ptxTxAux
-    saveTx (_ptxTxId, _ptxTxAux) `catches` handlers ack
-    addOnlyNewPendingTx ptx
-    when ack $ ptxUpdateMeta _ptxWallet _ptxTxId PtxMarkAcknowledged
+    now <- getCurrentTimestamp
+    if | PtxApplying poolInfo <- _ptxCond,
```

Comment:
Interestingly, this block of code from `108:116` was not part of https://github.com/input-output-hk/cardano-sl/pull/2270 -- Do you have any idea where this is coming from?

Reply:
If I see correctly:

[CSL-2135] Cancel PTX on resubmission after 1h limit

https://github.com/input-output-hk/cardano-sl/commit/fa3d120b4121dd8b72f3fd3ec629633785da7795

Reply:
Ah crap, and that was ported by you as well, but that PR is blocked by CI, effectively creating a bit of a deadlock.